client/src/components/Profile/ui/PrescreeningFeedback.tsx

```diff
@@ -3,65 +3,103 @@ import { StageInterviewDetailedFeedback } from '@common/models/profile';
 import { CODING_LEVELS, FeedbackStepId, SKILLS_LEVELS } from 'data/interviews/technical-screening';
 import { InterviewFeedbackStepData, InterviewFeedbackValues, InterviewQuestion } from '@common/models';
 import { Rating } from '@client/components/Rating';
+import React from 'react';
```

Comment:
Let's import only what we need  
```suggestion
import { useMemo } from 'react';
```